_posts/2022-04-07-launching-open-source-operator-for-openSearch.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Introducing Kubernetes Open-Source Operator for OpenSearch"
+authors:
+  - Opster
+date: 2022-04-07
+categories:
+  - technical-post
+twittercard:
+  description: "The Kubernetes OpenSearch Operator is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters."
+---
+
+
+Launching Open-Source Operator for OpenSearch 
+----------------------------------------------
+
+### With powerful capabilities for orchestration & management of OpenSearch clusters. The Operator is led by Opster with partners including AWS, SUSE Rancher, Maibornwolff, Logz.io and more.
```

Comment:
Seems repetitive - this is also covered in the `Overview` section. Can we remove this?